### ISSUE TITLE
Issue with "ARIA: button role": missing rendering of accessibility subnav

### ISSUE THREAD

**Issue by booc0mtaco** [mdn/content#3292]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role

#### What information was incorrect, unhelpful, or incomplete?

The sub-nav on each accessibility pages does not render.

#### Specific section or headline?

`<p>{{Page("/en-US/docs/Web/Accessibility/ARIA/Roles", "Subnav")}}</p>`

#### What did you expect to see?

- Some rendering of the sub-nav

#### Did you test this? If so, how?

Loaded https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/accessibility/aria/roles/button_role`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/accessibility/aria/roles/button_role/index.html
* Last commit: https://github.com/mdn/content/commit/8b12e7566336b6b4ee97d87e0889bf55a7d1dde5
* Document last modified: 2021-03-17T09:53:41.000Z

</details>

**Comment by chrisdavidmills** [comment]
This should no longer be there. Removed: https://github.com/mdn/content/commit/c358bf10987a1469196d976f5ba58b04752f694a

Thanks for reporting!

**Comment by booc0mtaco** [comment]
Thanks @chrisdavidmills . I noticed this same thing appears to show up on all the roles pages linked from here https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles . I am not too familiar with how the documents are generated, but would I expect there to be many edits to strip out all the instances?
